joomla/joomla-cms
# [5.2] Update NPM dependency micromatch

Pull Request for Issue # .

### Summary of Changes

As the title says.

### Testing Instructions

Check that the npm build and all integration and system tests pass in Drone.

Check that the packages step passes in Drone, so downloads are available.

Run `npm ci`.

Then run `npm list micromatch`.

### Actual result BEFORE applying this Pull Request

`npm ci` reports 2 moderate severity vulnerabilities.

`npm list micromatch`:
```
joomla@5.2.0 /home/richard/lamp/public_html/joomla-cms-5.2-dev
└─┬ stylelint@14.16.1
  ├─┬ fast-glob@3.3.2
  │ └── micromatch@4.0.7 deduped
  └── micromatch@4.0.7
```

### Expected result AFTER applying this Pull Request

`npm ci` reports only 1 moderate severity vulnerability. Besides this it shows the same result as without this PR.

`npm list micromatch`:
```
joomla@5.2.0 /home/richard/lamp/public_html/joomla-cms-5.2-dev
└─┬ stylelint@14.16.1
  ├─┬ fast-glob@3.3.2
  │ └── micromatch@4.0.8 deduped
  └── micromatch@4.0.8
```

### Link to documentations
Please select:
